docs/howto/index.txt

```diff
@@ -20,7 +20,6 @@ you quickly accomplish common tasks.
    upgrade-version
    error-reporting
    initial-data
-   jython
```

Comment:
I should point out that Tim had reservations about breaking links to the documentation. See [comment](https://code.djangoproject.com/ticket/28954#comment:9) on the ticket.

At the same time, we shouldn't promote things that will not be supported. If we're that worried about broken links in the dev version of the documentation we could still remove the bit in `docs/topics/install.txt` and restore this page but just flat out say it is no longer supported.

Reply:
IMO, the dev docs are by definition unstable. In my mind, it is like linking to a file on the master branch of a GitHub project. If people want the guarantee than an article exists, they should link to a version of the docs. So -- just my opinion -- I think this is okay to drop.

Reply:
OK, yes. I think both `dev` and `stable` imply a moving target. (vs specific versions, which won't break). Let's have this. Thanks both.